src/libs/SidebarUtils.js

```diff
@@ -296,7 +296,7 @@ function getOptionData(reportID) {
     result.text = reportName;
     result.subtitle = subtitle;
     result.participantsList = participantPersonalDetailList;
-    result.icons = ReportUtils.getIcons(report, personalDetails, policies, personalDetail.avatar);
+    result.icons = ReportUtils.getIcons(report, personalDetails, policies, ReportUtils.getDefaultAvatar(personalDetail.login));
```

Comment:
Just checking, we only want to get default avatars here? I was guessing we'd want to maybe use `getCorrectAvatar` here instead?